src/Properties/ModelPropertyHelper.php

```diff
@@ -92,16 +92,22 @@ public function getDatabaseProperty(ClassReflection $classReflection, string $pr
     {
         try {
             /** @var Model $modelInstance */
-            $modelInstance = $classReflection->getNativeReflection()->newInstanceWithoutConstructor();
+            $modelInstance = $classReflection->getNativeReflection()->newInstance();
```

Comment:
The constructor is skipped to avoid all the db initialization logic---I'm not sure how the tests are passing because this is failing for me locally with:

```
Message:  Database hosts array is empty. (Connection: mysql, SQL: select column_name as `name`, data_type as `type_name`, column_type as `type`, collation_name as `collation`, is_nullable as `nullable`, column_default as `default`, column_comment as `comment`, generation_expression as `expression`, extra as `extra` from information_schema.columns where table_schema = ':memory:' and table_name = 'guarded_models' order by ordinal_position asc)
Location: ~/sources/laravel/larastan/vendor/laravel/framework/src/Illuminate/Database/Connectors/ConnectionFactory.php:210
```